### DISCUSSION TITLE
Why isn't there a complete guide for developers?

### DISCUSSION THREAD

**Discussion by asemandeveloper** [ToolJet/ToolJet#11322] [General]
Hello everyone
I use Docker on Windows 11. I want to upload the source code of ToolJet and make changes in it and develop it. Unfortunately, I did not find a suitable guide for this. Setting the .env parameters for this is unclear and there is no accurate and proper explanation.
Please help and provide an .env file with proper settings.
thanks

**Comment by brahma-dev** [comment]
I am not from the team, but this might help : https://docs.tooljet.com/docs/contributing-guide/setup/docker/

**Reply by asemandeveloper** [reply]
I followed the instructions. It worked fine. However, only three moving dots are displayed in the browser.

**Comment by PriteshKiri** [comment]
Hey @asemandeveloper ,

Can you please drop your query in the general channel of our [Slack](https://www.tooljet.com/slack) community?

Our team would be happy to assist you there.
